client/src/theme.ts

```diff
@@ -3,9 +3,10 @@ import { DefaultTheme } from 'styled-components';
 export const myTheme: DefaultTheme = {
     colors: {
         blue: '#2259B1',
-        yellow: '#D4DB2E',
+        lemon: '#D4DB2E',
```

Comment:
Why `lemon`?

Reply:
I found the name of that color on the Internet :))